docs/docs-content/clusters/cluster-management/compliance-scan.md

```diff
@@ -24,6 +24,15 @@ the row of the node you want to scan, and select **Turn off maintenance mode**.
 
 :::
 
+## Airgap Environment
+
+If you are using self-hosted Palette or VerteX in an airgap environment, you must ensure the conformance packs are
+available in the self-hosted OCI registry. Refer to the
```

Comment:
you must ensure the conformance packs are available in the self-hosted OCI registry before you trigger scans or backup.